.github/workflows/rebuild-templates.yml

```
@@ -113,41 +193,39 @@ jobs:
           ssh-key: ${{ secrets.PROXMOX_SSH_KEY }}
           proxmox-host: ${{ secrets.PROXMOX_SSH_HOST }}
 
-      - name: Clean stale VM at target VMID
+      - name: Clean stale VM at staging VMID
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2029:info:4:16: Note that, unescaped, this expands on the client side

(shellcheck)